### PR TITLE
Login page responsive

### LINKED ISSUE #36241: Login page is not responsive.

### PR DESCRIPTION
**Pull request by mattgoud**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | login page responsive
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Go to BO login page
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/9367542326
| Fixed issue or discussion?     | Fixes #36241 
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (nicosomb) [inline .vscode/settings.json]: @mattgoud please remove this file ;-) it's dependent from your IDE configuration and we don't commit this kind of files in the repository.

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (mattgoud) [reply .vscode/settings.json]: ok @nicosomb done

**Review by boherm — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

**Review by jolelievre — APPROVED** [review]

**Comment by jolelievre** [comment]
Thanks @mattgoud

**Pull request merged**